glide.lock

```diff
@@ -1,8 +1,8 @@
 hash: f70d8455d9f5f68d4449263ab8d0485d5b938b65b4c31716be50aac64fedc6ae
```

Comment:
Without `glide.yaml`, the next time someone runs `glide up`, it'll lose the new zglob dependency.